### PR TITLE
DOC-10895 Cloud 2.0 update export logs and metrics for Azure Monitor

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-10895

- In cockroachdb-feature-availability.md, (1) replaced dedicated with advanced, (2) updated links to export-*-advanced, (3) replaced AWS CloudWatch with Amazon CloudWatch.
- In export-metrics.md, (1) removed mentions of Azure Monitor.
- In export-metrics-advanced.md, (1) added mentions of Azure Monitor, (2) replaced mentions of dedicated with advanced, (3) added Prometheus preview note for Azure.
- In export-logs.md, (1) removed mentions of Azure monitor, (2) added filter-tabs include for Standard/Advanced, (3) resolved merge conflict.
- In export-logs-advanced.md, (1) added mentions of Azure monitor, (2) replaced AWS CloudWatch with Amazon CloudWatch, (3) corrected references to config step.
- In export-*.md files, (1) replaced cockroachlabs.com links with liquid links, (2) replaced AWS CloudWatch with Amazon CloudWatch, (3) removed en-us from external links.

### CODE REVIEW

**Review by kathancox — APPROVED** [review]
Just some suggestions, otherwise LGTM

File: src/current/cockroachcloud/export-metrics-advanced.md [hunk 1]
```diff
@@ -328,10 +337,10 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
...
+Exporting metrics to Azure Monitor is available only on CockroachDB {{ site.data.products.advanced }} clusters that are hosted on Azure. {{ site.data.products.advanced }} clusters hosted in AWS If your CockroachDB {{ site.data.products.advanced }} cluster is hosted on AWS, you can export metrics to [Amazon CloudWatch]({% link cockroachcloud/export-metrics-advanced.md %}?filters=aws-metrics-export#enable-metrics-export). If it is hosted on GCP, you can export metrics to [Datadog]({% link cockroachcloud/export-metrics-advanced.md %}?filters=datadog-metrics-export#enable-metrics-export) or [Prometheus]({% link cockroachcloud/export-metrics-advanced.md %}?filters=prometheus-metrics-export#enable-metrics-export) instead.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-advanced.md]: thanks for catching that, removed.

File: src/current/cockroachcloud/export-metrics-advanced.md [hunk 2]
```diff
@@ -350,14 +359,14 @@ To enable metrics export to Azure Monitor:
     cat ca.crt ca.key | awk '{printf "%s\\n", $0}'  > concatenated.pem
     ~~~
 
-1. To give access to the CockroachDB cluster to your Azure tenant, [create a Microsoft Entra application](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal). When registering the application, leave the optional **Redirect URI** empty since certificate-based authentication is being used, not browser-based authentication.
+1. To give access to the CockroachDB cluster to your Azure tenant, [create a Microsoft Entra application](https://learn.microsoft.com/entra/identity-platform/howto-create-service-principal-portal). When registering the application, leave the optional **Redirect URI** empty since certificate-based authentication is being used, not browser-based authentication.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-advanced.md]: good clarification. updated.

File: src/current/cockroachcloud/export-metrics-advanced.md [hunk 3]
```diff
@@ -350,14 +359,14 @@ To enable metrics export to Azure Monitor:
...
   1. From the **Overview** page for the Entra application, note the values for **Display Name**, **Application (client) ID**, and **Directory (tenant) ID**, which will be used in step 5.
 
-1. To allow Azure Monitor to receive metrics, [create an Application Insights resource](https://learn.microsoft.com/en-us/azure/azure-monitor/app/create-workspace-resource).
+1. To allow Azure Monitor to receive metrics, [create an Application Insights resource](https://learn.microsoft.com/azure/azure-monitor/app/create-workspace-resource).
 
-  1. In the newly-created Application Insights resource, add a role assignment that assigns the Entra application (search for the **Display Name** from step 3) to the role [Monitoring Metrics Publisher](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/monitor#monitoring-metrics-publisher).
+  1. In the newly-created Application Insights resource, add a role assignment that assigns the Entra application (search for the **Display Name** from step 3) to the role [Monitoring Metrics Publisher](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles/monitor#monitoring-metrics-publisher).
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-advanced.md]: updated

**Review by florence-crl — COMMENTED** [review]
TFTRs

**Pull request merged**